packages/lib/src/components/Contact/components/ContactForm.tsx

```diff
@@ -44,22 +46,28 @@ export function ContactForm({ formState, control }: Props): JSX.Element {
   const { t } = useTranslation()
 
   return (
-    <Card shadow="sm" p="lg" radius="md" withBorder role="form">
-      <Title order={3} mb="md">
+    <Card
+      shadow="sm"
+      radius="md"
```

Comment:
Shadow and radius can also be outsourced (please check other occurrences as well)